README.md

```diff
@@ -62,7 +63,7 @@ This will serve as a central hub to store all community made themes
 If you have an issue, raise an Issue with the theme name in the title i.e. `[Light] Side Nav not readable on mobile` and tag the person from the Readme table. 
 Kavita team does not provide support on any themes, even those created by majora2007. 
 
-If you identify bugs in the themeing, please raise an Issue on the Kavita Github for support or development work. Theme feature requests belong on the Kavita Discussions. 
+If you identify bugs in custom themes, please raise an Issue on the Kavita Github for support or development work. Theme feature requests belong on the Kavita Discussions. 

```

Comment:
Please revert this. This statement is for if the css variables are not working, missing, etc and require a Kavita change.

Custom theme issues should be created on this repo and tag the creator as Kavita does not support bugs on custom themes.

Reply:
No problem. I just committed again with that line reverted.